app.js

```diff
@@ -143,40 +143,42 @@ app.controller('Damage', function ($scope) {
 
   $scope.stringifyUrlData = function () {
     var data = {
-      increase: $scope.sections.monster.tables.increase.values.slice(0, -1),
-      more: $scope.sections.monster.tables.more.values.slice(0, -1),
-      shift: $scope.sections.shift.tables.shifts.values.slice(0, -1),
-      armor: $scope.sections.mitigation.armor,
-      charges: $scope.sections.mitigation.charges,
-      reduction: $scope.sections.mitigation.tables.reduction.values.slice(0, -1),
-      flat: $scope.sections.taken.tables.flat.values.slice(0, -1),
+      i: $scope.sections.monster.tables.increase.values.slice(0, -1),
+      m: $scope.sections.monster.tables.more.values.slice(0, -1),
+      s: $scope.sections.shift.tables.shifts.values.slice(0, -1),
+      a: $scope.sections.mitigation.armor,
+      c: $scope.sections.mitigation.charges,
+      r: $scope.sections.mitigation.tables.reduction.values.slice(0, -1),
+      f: $scope.sections.taken.tables.flat.values.slice(0, -1),
       reduced: $scope.sections.taken.tables.reduced.values.slice(0, -1),
```

Comment:
if we're gonna go this route, might as well do something like d for reduced, and t for resist or w/e